### DISCUSSION TITLE
Is there a way to group by a column without an aggregated row?

### DISCUSSION THREAD

**Discussion by oliwheeler** [TanStack/table#5730] [Q&A]
In the image below, the data is grouped by **Region** but there is no aggregated row.
Is there a way to do this in TanStack table? 

Thank you

![pivot table](https://www.pivot-table.com/wp-content/uploads/2017/04/pivotitemshidden06_thumb.png)

**Comment by jmtimko5** [comment]
@tannerlinsley would you be open to me raising a PR to allow for multiple `row.groupingColumnId` values? I'd like to get this sort of display formatting enabled for cases when you have several grouping dimensions. The vertical real estate taken up by the several placeholder cells gets pretty unwieldy fast (see screenshot for example).

<img width="524" alt="image" src="https://github.com/user-attachments/assets/db52933e-7b5b-40a1-b7af-1bec3d1d18f6">
